include/items.php

```diff
@@ -502,6 +502,8 @@ function item_store($arr, $force_parent = false, $notify = false, $dontcache = f
 				$arr['guid'] = uri_to_guid($arr['uri'], $a->get_hostname());
 			}
 		}
+	} else {
+		$arr['network'] = trim(defaults($arr, 'network', NETWORK_PHANTOM));
```

Comment:
This could conflict with this line: https://github.com/friendica/friendica/blob/develop/include/items.php#L647

Because of this I wouldn't like to set a default value here.